queries.sql

```diff
@@ -29,4 +29,109 @@ WHERE
 
 SELECT * from animals
 WHERE
-    weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+    weight_kg BETWEEN 10.4 and 17.3;
+
+-- Inside a transaction update the animals table by setting the species column 
+-- to unspecified. Verify that change was made. Then roll back the change and 
+-- verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * from animals
+
+ROLLBACK;
+
+SELECT * from animals
+
+-- Inside a transaction:
+-- Update the animals table by setting the species column to digimon for all 
+-- animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all 
+-- animals that don't have species already set.
+-- Verify that changes were made.
+-- Commit the transaction.
+-- Verify that changes persist after commit.
+-- Take a screenshot of the results of your actions.
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name like '%mon%';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * from animals;
+
+COMMIT;
+
+SELECT * from animals
+
+BEGIN;
+DELETE from animals;
+
+SELECT * from animals;
+
+ROLLBACK;
+
+SELECT * from animals
+
+-- Inside a transaction:
+-- Delete all animals born after Jan 1st, 2022.
+-- Create a savepoint for the transaction.
+-- Update all animals' weight to be their weight multiplied by -1.
+-- Rollback to the savepoint
+-- Update all animals' weights that are negative to be their weight 
+-- multiplied by -1.
+-- Commit transaction
+-- Take a screenshot of the results of your actions.
+
+BEGIN;
+DELETE FROM animals 
+WHERE date_of_birth > '2022-01-01';
+
+SELECT * from animals;
+
+SAVEPOINT sp1;
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+
+SELECT * from animals;
+
+ROLLBACK TO SAVEPOINT sp1;
+
+UPDATE animals
+SET weight_kg = weight_kg* (-1)
+WHERE weight_kg < 0;
+
+SELECT * from animals;
+
+Commit;
+
+SELECT * from animals
+
+-- ////Picture 4///
+
```

Comment:
- [ ] Great work on making the screenshots but i guess you misunderstood the requirement here because the requirement is to add all the screenshots in the pull request descriptions means the screenshots themselves 🤔 So kindly add all the screenshots in the pull request description to make your work follow the project requirements 👍